src/udiskslinuxblock.c

```diff
@@ -3445,6 +3452,7 @@ udisks_linux_block_handle_format (UDisksBlock             *block,
   g_variant_lookup (options, "encrypt.iterations", "u", &encrypt_iterations);
   g_variant_lookup (options, "encrypt.time", "u", &encrypt_time);
   g_variant_lookup (options, "encrypt.threads", "u", &encrypt_threads);
+  g_variant_lookup (options, "encrypt.label", "&s", &encrypt_label);
```

Comment:
Would you document this new argument in the D-Bus docs please?

Reply:
I knew I forgot something. Fixed.

Reply:
Fixed again :man_facepalming: